tapestry.js

```diff
@@ -17,7 +17,8 @@ const // declared
     COLOR_LINK = "#999",
     COLOR_SECONDARY_LINK = "transparent",
     CSS_OPTIONAL_LINK = "stroke-dasharray: 30, 15;",
-    FONT_ADJUST = 1.25;
+    FONT_ADJUST = 1.25,
+    GET_TAPESTRY_URL = "http://localhost:8888/tapestry-wp/wp-json/tapestry-tool/v1/tapestries/" + tapestryWpPostId;
```

Comment:
Could you please put the URL for the API in a separate constant so that we can use it within multiple endpoints without needing to add in the URL every time?

Reply:
In fact, please pass the URL from the template file instead so that we can get it dynamically rather than typing it in.

Reply:
URL is passed from the template file now. The D3 will just use the correct endpoint.